### PR TITLE
EPMRPP-35675: add launch description

### PR DESCRIPTION
**Pull request by Andrew-Omelchenko**

### CODE REVIEW

File: app/src/pages/inside/common/infoLine/attributes/attributesTooltip/attributesTooltip.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <div className={cx('attributes-tooltip')}>
+    {attributes.length > 0 ? (
+      attributes.map((item) => (
+        <div key={`${item.key}-${item.value}`}>
+          <span>{item.key}:&nbsp;</span>
+          <span>{item.value}</span>
+        </div>
+      ))
+    ) : (
+      <FormattedMessage id="AttributesTooltip.message" defaultMessage="No attributes" />
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/infoLine/attributes/attributesTooltip/attributesTooltip.jsx]: I think it's better to hide the icon if an item doesn't have attributes. To be consistent with grid items.

File: app/src/pages/inside/common/infoLine/description/descriptionTooltip/descriptionTooltip.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export const DescriptionTooltip = ({ description }) => (
+  <div>
+    {description ? (
+      <div className={cx('description-tooltip')}>
+        <ScrollWrapper>
+          <MarkdownViewer value={description} />
+        </ScrollWrapper>
+      </div>
+    ) : (
+      <FormattedMessage id="DescriptionTooltip.message" defaultMessage="No description" />
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/infoLine/description/descriptionTooltip/descriptionTooltip.jsx]: It's better to hide the icon to to have more consistent behavior with grid items

File: app/src/pages/inside/common/durationBlock/durationBlock.jsx [hunk 3]
```diff
@@ -83,12 +83,16 @@ export class DurationBlock extends Component {
     }).isRequired,
     status: PropTypes.string,
     itemNumber: PropTypes.number,
+    iconStyle: PropTypes.string,
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/durationBlock/durationBlock.jsx]: iconClass

File: app/src/pages/inside/common/durationBlock/durationBlock.jsx [hunk 4]
```diff
@@ -83,12 +83,16 @@ export class DurationBlock extends Component {
...
+    durationStyle: PropTypes.string,
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/durationBlock/durationBlock.jsx]: durationClass

File: app/src/pages/inside/common/infoLine/attributes/attributesTooltip/attributesTooltip.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import styles from './attributesTooltip.scss';
+
+const cx = classNames.bind(styles);
+
+export const AttributesTooltip = ({ attributes }) => (
+  <div className={cx('attributes-tooltip')}>
+    {attributes.length > 0 ? (
+      attributes.map((item) => (
+        <div key={`${item.key}-${item.value}`}>
+          <span>{item.key}:&nbsp;</span>
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/infoLine/attributes/attributesTooltip/attributesTooltip.jsx]: `import { formatAttribute } from 'common/utils/attributeUtils';`

**Comment by Andrew-Omelchenko** [comment]
Fixed.

**Pull request merged**